test_workloads.py

```diff
@@ -274,6 +274,14 @@ def test_tfserving_workload(self):
         assert("Success" in tfserving_contents)
         assert("error: " not in tfserving_contents)
 
+    
+	@pytest.mark.examples
```

Comment:
If for some reason, it is being enabled for only specific distro, we need to update the condition here as well